desk/app/activity.hoon

```diff
@@ -521,6 +570,15 @@
 ++  base
   ^-  index:a
   (~(got by indices) [%base ~])
+++  give-update
+  |=  [=update:a path=(unit path)]
+  ^+  cor
+  =/  v0-paths  ?~(path ~[/ /v0] ~[/ /v0 u.path (weld /v0 u.path)])
+  =/  v0-cage=cage  activity-update+!>((update-0:convert-to update))
+  =/  v1-paths  ?~(path ~[/v1] ~[/v1 (weld /v1 u.path)])
+  =/  v1-cage=cage  activity-update-1+!>(update)
```

Comment:
```suggestion
  =/  v0-paths  ?~(path ~[/ /v0] ~[/ /v0 u.path [%v0 u.path]])
  =/  v0-cage=cage  activity-update+!>((update-0:convert-to update))
  =/  v1-paths  ?~(path ~[/v1] ~[/v1 [%v1 u.path]])
  =/  v1-cage=cage  activity-update-1+!>(update)
```

Don't actually need to weld these, can just prepend the item to the path through consing.